tests/sequence/test_sound_classes.py

```diff
@@ -17,12 +17,6 @@ def test_ipa2tokens(test_data):
     seq = 'ʰto͡i'
     assert len(ipa2tokens(seq)) == 2
 
-    seq = 'th o x t a'
```

Comment:
In this case, we may want to assert a value error, so one could use `assert raises(ValueError)`, explicit error handling, given that this is should be the new behavior, right?